app/views/transaction.py

```diff
@@ -0,0 +1,766 @@
+                 200:
+                     description: Account transaction details with this id updated successfully
+                     schema:
+                         AccountTransactionDetailsSchema
+         """
+        try:
+            # retrieve body data from input JSON
+            data = request.get_json()
+            errors = account_transaction_details_schema.validate(data, partial=True)
+            if errors:
+                logger.error("Missing or sending incorrect data to create an activity {}".format(errors))
+                response = ResponseGenerator(data={},
+                                             message=errors,
+                                             success=False,
+                                             status=HTTPStatus.BAD_REQUEST)
+                return response.error_response()
+
+            account_transaction_details_data = AccountTransactionDetails.query.filter(AccountTransactionDetails.id == account_transaction_details_id).first()
+            if not account_transaction_details_data:
+                raise AccountTransactionDetailsObjectNotFound("Account transaction details with this id does not exist")
+
+            account_transaction_details_data.transaction_amount = data.get('transaction_amount', account_transaction_details_data.transaction_amount)
+            account_transaction_details_data.bank_account_id = data.get('bank_account_id', account_transaction_details_data.bank_account_id)
+            account_transaction_details_data.transaction_type_id = data.get('transaction_type_id', account_transaction_details_data.transaction_type_id)
+            account_transaction_details_data.fund_transfer_id = data.get('fund_transfer_id', account_transaction_details_data.fund_transfer_id)
+            account_transaction_details_data.fund_transfer_type = data.get('fund_transfer_type', account_transaction_details_data.fund_transfer_type)
+
+            db.session.commit()
+            result = account_transaction_details_schema.dump(account_transaction_details_data)
+
+            logger.info("Response for put with id request for account transaction details {}".format(result))
+            response = ResponseGenerator(data=result,
+                                         message="Account transaction details with this id updated successfully",
+                                         success=True,
+                                         status=HTTPStatus.OK)
+            return response.success_response()
+
+        except AccountTransactionDetailsObjectNotFound as err:
+            logger.exception(err.message)
+            response = ResponseGenerator(data={},
+                                         message=err.message,
+                                         success=False,
+                                         status=HTTPStatus.NOT_FOUND)
+            return response.error_response()
+
+    def delete(self, account_transaction_details_id):
```

Comment:
We do not need delete transaction API ideally.

Reply:
Made the changes as suggested.